Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.9-slim-bullseye as base
+
+#Common needs
+#Set Working Directory
+WORKDIR /todo_app
+#install Poetry
+RUN pip install poetry
+#copy the config files
+COPY pyproject.toml poetry.toml poetry.lock ./
+#install dependencies
+RUN poetry install
+#Copy project files
+COPY . /
```

Comment:
I think you mean `COPY . ./`

If the path starts with a / that will copy things to the root of the Linux system. You want to copy into the current working directory.

But you would have seen the app still working because of the bind mount

Reply:
change made